pkgs/development/libraries/librealsense/default.nix

```diff
@@ -78,7 +78,10 @@ stdenv.mkDerivation rec {
   # script does not do this, and it's questionable if intel knows it should be
   # done
   # ( https://github.com/IntelRealSense/meta-intel-realsense/issues/20 )
-  postInstall = lib.optionalString enablePython  ''
+  postInstall = ''
+    substituteInPlace $out/lib/cmake/realsense2/realsense2Targets.cmake \
+    --replace "\''${_IMPORT_PREFIX}/include" "$dev/include"
```

Comment:
Why weren't the cmake files automatically moved to the dev output?

https://github.com/NixOS/nixpkgs/blob/1e08d91dfa9facdade906a99f52b4bcf9ac84c92/pkgs/build-support/setup-hooks/multiple-outputs.sh#L172

Reply:
The cmake files are moved to the dev output, however, the `_IMPORT_PREFIX` is set to $out/include:
```zsh
/nix/store/im4q79iymcm0f8nki4vyqw1wb4brnzh6-librealsense-2.45.0-dev/lib/cmake/realsense2
• ag IMPORT_PREFIX
realsense2Targets.cmake
50:set(_IMPORT_PREFIX "/nix/store/yv2838a3hfqd52qvpj1lrsk50h6q7svs-librealsense-2.45.0")
67:  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include"
83:set(_IMPORT_PREFIX)
```

Reply:
Upstreaming the fix will be hard, since librealsense is discontinued. The hardware still sees use in academia however.

Reply:
Their tracking camera line got discontinued, but they still support librealsense for the stereo cameras. The latest release was just 2 weeks ago. 
Even if the PR doesn't get merged, I just wanted to put the fix out there for anyone using librealsense with Nix.
imo Nix makes using dependency heavy libraries like this one a breeze, so thanks for all your work! 😀